test/extra/CMakeLists.txt

```diff
@@ -34,9 +34,9 @@ list(APPEND CMAKE_MODULE_PATH
 
 # Use target ID syntax if supported for AMDGPU_TARGETS
 if(TARGET_ID_SUPPORT)
-  set(AMDGPU_TARGETS gfx803;gfx900:xnack-;gfx906:xnack-;gfx908:xnack-;gfx1030:xnack- CACHE STRING "List of specific machine types for library to target")
+  set(AMDGPU_TARGETS set(AMDGPU_TARGETS gfx803;gfx900:xnack-;gfx906:xnack-;gfx908:xnack-;gfx1030 CACHE STRING "List of specific machine types for library to target")
```

Comment:
Looks like a typo here @eidenyoshida

Reply:
Good catch, I have another PR to fix some issues I made in this one